daemon/openrazer_daemon/hardware/headsets.py

```diff
@@ -303,3 +303,23 @@ def _resume_device(self):
             _dbus_chroma.set_breath_triple_effect(self, *args)
 
         self.disable_notify = False
+
+
+class RazerKrakenKittyEdition(__RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Kraken Kitty Edition
+    """
+    EVENT_FILE_REGEX = re.compile(
+        r'.*Razer_Kraken_Kitty_Chroma_Control-event-if00')
+
+    USB_VID = 0x1532
+    USB_PID = 0x0F19
+    METHODS = ['get_device_type_headset',
+               'set_none_effect', 'set_static_effect', 'set_breath_random_effect', 'set_breath_single_effect',
+               'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row',
+               'set_brightness', 'get_brightness']
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 4]
+
+    DEVICE_IMAGE = "https://assets3.razerzone.com/C2Ol5yRpYX8kE8JX3R9K2svFNaM=/1500x1000/https%3A%2F%2Fhybrismediaprod.blob.core.windows.net%2Fsys-master-phoenix-images-container%2Fhda%2Fh57%2F9055450103838%2FKraken-Kitty-Edition-Black-gallery-Hero.jpg"
```

Comment:
Maybe use https://hybrismediaprod.blob.core.windows.net/sys-master-phoenix-images-container/hda/h57/9055450103838/Kraken-Kitty-Edition-Black-gallery-Hero.jpg directly; it's interesting how many different subdomains and media hosts Razer uses